test/src/net/test_conn.c

```diff
@@ -9,10 +9,10 @@
 
 #define MAX_RESULT_SIZE	2048
 
-TS_FUNCTION_INFO_V1(test_conn);
+TS_FUNCTION_INFO_V1(ts_test_conn);
 
 Datum
-test_conn(PG_FUNCTION_ARGS)
+ts_test_conn(PG_FUNCTION_ARGS)
```

Comment:
Strictly, our test code doesn't need the `ts_` prefix, but I guess it doesn't hurt for consistency.

Reply:
That's what I figured.lso @cevian is talking